packages/client/tests/tests-type-cases.rs

```diff
@@ -0,0 +1,953 @@
+use std::collections::HashMap;
```

Comment:
this is really awesome - thank you for pushing this! a thought that came to me is that it would probably make sense to create a file per test? so we don't have a file with +1k lines, but rather if a user would like to see an example of interacting with n-type it can find it easier? feel free to ignore tho

Reply:
sure thing, i'll refactor